consensus/src/epoch_manager.rs

```diff
@@ -308,14 +308,17 @@ impl EpochManager {
             Ok(()) => { /* Expected */ }
             Err(Error::ValidatorKeyNotFound(error)) => {
                 error!(
+                    epoch = epoch,
                     error = error,
-                    "Unable to find the validator consensus key during initialize",
+                    "Unable to find the validator consensus key during initialize() and will not participate in consensus.\
+                     Will continue syncing until the validator consensus key matches in a future epoch."
```

Comment:
While this is an interesting message an all, I'm not convinced on its utility since we'll see a ton of errors and warnings afterward, so they'll have to look at the run books for consensus and safety rules anyway...

Reply:
@aching, I'll let you and @davidiw battle it out 😃 My opinion is similar to David's in that even if this error message is super lengthy/explicit, a non-specialist (e.g., operator), would still need to follow the runbook to know how to react, so I'm not sure we gain a whole lot. Thoughts?

Reply:
code as runbook! :trollface:

Reply:
Run books are great but ideally an error can provide some context to be standalone in describing the issue and what to do, even if that is just to point to a runbook.  That's kind of what error codes are for anyway.  I think this would be helpful for someone just skimming errors to figure out how much to panic and why.  It's good for me.